index.html

```diff
@@ -112,159 +111,615 @@ <h1><a href="/">Latest funding rounds on Index.co ></a></h1>
+                    </div>
                 </div>
+                <div class="lastest-news-container">
+                    <div class="latest-news-flex">
+                        <div class="latest-news-container-image">
+                            <a href="/">
+                                <img src="images/work-office-startup.png" alt="future business"
+                                    class="latest-news-image-size">
+                            </a>
+                        </div>
+                        <div class="latest-news-container-text">
+                            <a href="/" class="latest-news-container-text-bold">Future-proofing your business isn't
+                                about tech — key elements</a>
+                            <p><a href="/" class="latest-news-container-text-black">Michel Koopman </a>- 21 hours ago
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="lastest-news-container">
+                    <div class="latest-news-flex">
+                        <div class="latest-news-container-image">
+                            <a href="/">
+                                <img src="images/new-iOS-icons.png" alt="new iOS icons" class="latest-news-image-size">
+                            </a>
+                        </div>
+                        <div class="latest-news-container-text">
+                            <a href="/" class="latest-news-container-text-bold">These new iOS icons may give us a
+                                glimpse of Apple’s language</a>
+                            <p><a href="/" class="latest-news-container-text-black">Callum Booth </a>- 21 hours ago</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
         </section>
-   </main>
-   <footer class="footer-mobile">
-        <nav>
-            <div class="silver-black">
-                <div class="icons">
+        <!--hard-fork-section-->
+        <section class="hard-fork">
+
+            <div class="hard-fork-container">
+                <a class="hard-fork-container-tech" href="/">
+                    <h1>Tech ></h1>
+                </a>
+                <div class="hard-fork-1">
+                    <p class="span-1">MOST POPULAR</p>
+                    <a class="a-1" href="/">
+                        <h3>Intel's new AI helps you get just the right amount <br>of hate speech in your game chat</h3>
+                    </a>
+                    <p class="span-a"><a href="/">Ivan Mehta</a>- 1 day ago</p>
+                    <div class="hard-fork-latest">
+                        <img src="images/coinbase.jpg" alt="coinbase" class="hard-fork-image-22">
+                        <p class="hard-fork-p-a-1">LATEST</p>
+                        <a class="hard-fork-p-a-2" href="/">
+                            <p>What we know about Coinbase’s<br>April 14 IPO</p>
```

Comment:
Make this p-tag a heading tag (hx). Apply the change to the other p tags in hard-fork-p-a-2 classes too.